packages/fx-core/src/common/templatesUtils.ts

```diff
@@ -115,9 +115,9 @@ export function renderTemplateContent(
   filePath: string,
   data: Buffer,
   variables: { [key: string]: string }
-): string {
+): string | Buffer {
   if (path.extname(filePath) === templateFileExt) {
     return Mustache.render(data.toString(), variables);
   }
-  return data.toString();
+  return data;
```

Comment:
Tab template contains .png file, `toString()` breaks the picture.

Reply:
please add this info as a comment of the code. It is important.